packages/@remirror/core/src/builtins/persistent-selection-extension.ts

```diff
@@ -0,0 +1,50 @@
+// XXX: @remirror/core exports a _type_ 'DecorationSet', but we need access to the static methods as well.
+// See https://github.com/iview/iview/issues/4465 for a similar issue
+import { ExtensionPriority } from '@remirror/core-constants';
+import type { Static } from '@remirror/core-types';
+import { Decoration, DecorationSet } from '@remirror/pm/view';
+
+import { extensionDecorator } from '../decorators';
+import { PlainExtension } from '../extension';
+import type { CreatePluginReturn } from '../types';
+
+export interface PersistentSelectionOptions {
+  persistentSelectionClass?: Static<string>;
+}
+
+@extensionDecorator<PersistentSelectionOptions>({
+  defaultOptions: {
+    persistentSelectionClass: undefined,
```

Comment:
I actually feel like there should a default value here. Something like `selected-text`.

Reply:
I initially had a default, but that will break compatibility (as many of the unit tests show). 

Of course, the question is whether 100% compatibility is relevant here in the context of remirror.

Reply:
Oh, that's a valid point. 

I don't mind updating the snapshots, though. It's a core piece of functionality and my feeling is that it will be an expected behaviour for the majority of users. Also, the class is pretty harmless by itself, unless styles are assigned.